.gitignore

```diff
@@ -130,3 +130,5 @@ example/venv
 /example/pandora-linux-precompiled-v0.9.0.gz
 
 example/pandora-linux-precompiled-pandora_paper_tag1
+
+example/pandora-linux-precompiled-v0.9.0-rc1
```

Comment:
```suggestion
example/pandora-linux-precompiled*
```

Reply:
I actually ended up cleaning this `.gitignore` file as there were so many outdated entries. Could you please re-review?